SoS/SoS.tp2

```diff
@@ -1315,58 +1344,25 @@ BEGIN @21011
             ~%MOD_FOLDER%/Wed/Alt/AR4230N.WED~ ~override~
 
     COPY    ~%MOD_FOLDER%/Are/Alt/AR4230.are~ ~override~
-		GET_OFFSET_ARRAY regions_array ARE_V10_REGIONS 
-		PATCH_PHP_EACH regions_array AS num => offset BEGIN
-			READ_ASCII offset ~trigger_name~
-			PATCH_IF (~%trigger_name%~ STR_EQ ~Inf3304b~) BEGIN
-				SAY offset+0x64 @20075 // Social Club
-			END
-			PATCH_IF (~%trigger_name%~ STR_EQ ~Inf3351a~) BEGIN
-				SAY offset+0x64 @20076 // Narwhal
-			END
-			PATCH_IF (~%trigger_name%~ STR_EQ ~Info3301~) BEGIN
-				SAY offset+0x64 @20077 // Barracks
-			END
-			PATCH_IF (~%trigger_name%~ STR_EQ ~Info3303~) BEGIN
-				SAY offset+0x64 @20078 // Town hall
-			END
-			PATCH_IF (~%trigger_name%~ STR_EQ ~Info3307~) BEGIN
-				SAY offset+0x64 @20079 // Reading room
-			END
-			PATCH_IF (~%trigger_name%~ STR_EQ ~Inf3345~) BEGIN
-				SAY offset+0x64 @20771 // The Splintered Stair
-			END
-			PATCH_IF (~%trigger_name%~ STR_EQ ~Inf3301~) BEGIN
-				SAY offset+0x64 @20077 // Barracks
-			END
-		END
-		GET_OFFSET_ARRAY auto_map_array 0xc4 4 0xc8 4 0 0 0x34 
-		PATCH_PHP_EACH auto_map_array AS num => offset BEGIN
-			READ_SHORT offset xcoord
-			PATCH_IF xcoord = 4109 BEGIN
-				SAY offset+0x04 @20081 // South to Chionthar
-			END
-			PATCH_IF xcoord = 2729 BEGIN
-				SAY offset+0x04 @20078 // Town hall
-			END
-			PATCH_IF xcoord = 4444 BEGIN
-				SAY offset+0x04 @20077 // Barracks
-				WRITE_SHORT offset      4850 // Barracks X
-				WRITE_SHORT offset+0x02 3415 // Barracks Y
-			END
-			PATCH_IF xcoord = 93 BEGIN
-				SAY offset+0x04 @20075 // Social Club
-			END
-			PATCH_IF xcoord = 2218 BEGIN
-				SAY offset+0x04 @20079 // Reading room
-			END
-			PATCH_IF xcoord = 830 BEGIN
-				SAY offset+0x04 @20080 // North Lyrar's Hold
-			END
-			PATCH_IF xcoord = 1721 BEGIN
-				SAY offset+0x04 @20076 // Narwhal
-			END
-		END
+        // Info points
+        SAY 0x257c @20075 // Social Club
```

Comment:
Is it intended to restore a more "basic" way to assign info triggers?

Reply:
I have to delete the pull request again, something went wrong :(